lib/generators/solidus_starter_frontend/rspec_generator.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module SolidusStarterFrontend
+  class RspecGenerator < Rails::Generators::Base
+    source_root File.expand_path('../../..', __dir__)
+
+    def install
+      # solidus_dev_support 2.5.0 depends on this puma version.
+      gsub_file 'Gemfile', /gem ['"]puma['"].*/, "# gem 'puma' # solidus_dev_support 2.5.0 depends on puma ~> 4.3. See puma declaration below."
```

Comment:
It's quite unfortunate that we need to do that, even more considering that `solidus_dev_support` is not a dependency on production. I bet that we could update the `puma` version on `solidus_dev_support` without issues. We could do that as a follow-up and then open another PR updating from `solidus_starter_frontend`. Or, at least, we should open an issue on `solidus_dev_support` so that we don't lose track of it.

Reply:
@waiting-for-dev Okay, I'll create the tickets. TODO.